src/fs/mod.rs

```diff
@@ -1,5 +1,6 @@
 #[cfg(all(feature = "fuse", feature = "pci"))]
 pub(crate) mod fuse;
+mod fuse_abi;
```

Comment:
The issue with Clippy is that with `--no-default-features`, the constant is unused.

Would something like this work?

```suggestion
#[cfg(all(feature = "fuse", feature = "pci"))]
mod fuse_abi;
```